chloggen/internal/config/config.go

```diff
@@ -29,13 +29,16 @@ const (
 	DefaultTemplateYAML      = "TEMPLATE.yaml"
 	DefaultChangeLogKey      = "default"
 	DefaultChangeLogFilename = "CHANGELOG.md"
+
+	DefaultValidationMode = "none"
 )
 
 type Config struct {
 	ChangeLogs        map[string]string `yaml:"change_logs"`
 	DefaultChangeLogs []string          `yaml:"default_change_logs"`
 	EntriesDir        string            `yaml:"entries_dir"`
 	TemplateYAML      string            `yaml:"template_yaml"`
+	ComponentPrefixes []string          `yaml:"component_prefixes"`
```

Comment:
Any docs for it? I am guessing that it should be documented here: `chloggen/internal/config/config.yaml`

Reply:
Good point, added some